ServerlessLibraryUI/src/components/AppTopbar.vue

```diff
@@ -28,7 +28,8 @@
         <select class="c-search dropdown2"  v-model="language">
           <option value="" selected >Language: All</option>
           <option value="javascript">JavaScript</option>
-          <option value="csharp">C#</option>
+          <option value="python">Python</option>
+          <option value="java">Java</option>
         </select>
```

Comment:
You should also update the ToDisplayLanguage filter in AppItem.vue

Reply:
Good catch! Done.